ethereum_history_api/oracles/src/script/prepareNoirFixtures.ts

```diff
@@ -3,6 +3,10 @@ import { writeFile, mkdir, rm } from 'fs/promises';
 import { createHeaderFixture } from './noir_fixtures/header.js';
 import { createStateProofFixture } from './noir_fixtures/state_proof.js';
 import { createAccountFixture } from './noir_fixtures/account.js';
+import { createStorageProofFixture } from './noir_fixtures/storage_proof.js';
+
+const USDC_TOKEN_CONTRACT_ADDRESS = '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48';
```

Comment:
Nice!

Reply:
But now it's inconsistent with other fixtures.
I would choose one of two options:
1. Give names variable only where it's not obvious from the context - what this address is. Here - only the `first` is not obvious.
2. Give names to all

Reply:
OK, only `CIRCLE_USDC_BALANCE_STORAGE_KEY` has a name and token contract address should be obvious from context